test/test_urdf.py

```diff
@@ -181,6 +181,30 @@ def test_link_multiple_collision(self):
 </robot>'''
         self.parse_and_compare(xml)
 
+    def test_version_attribute_int(self):
+        xml = '''<?xml version="1.0"?>
+<robot name="test" version="1">
+</robot>'''
+        self.parse_and_compare(xml)
+
+    def test_version_attribute_float(self):
+        xml = '''<?xml version="1.0"?>
+<robot name="test" version="1.0">
+</robot>'''
+        self.parse_and_compare(xml)
+
+    def test_version_attribute_invalid(self):
+        xml = '''<?xml version="1.0"?>
+<robot name="test" version="foo">
+</robot>'''
+        #self.assertRaises(xmlr.core.ParseError, self.parse, xml)
```

Comment:
can this commented line be deleted?

Reply:
Yeah, definitely, just a leftover from testing.  Done in 9f78b7f